tests/debugpy/server/test_cli.py

```diff
@@ -71,7 +82,7 @@ def test_targets(cli, target_kind, mode, address, wait_for_client, script_args):
     expected_options = {
         "mode": mode,
         "target_kind": target_kind,
-        "wait_for_client": bool(wait_for_client),
+        "wait_for_client": False
```

Comment:
This seems off? The parameter is being skipped now?

Reply:
This was just a refactor with no functional difference. It defaults to false, so in the case of the empty string, the conditional on line 96 will eval to false. But in the case where it has a value, line 96 will eval to true, and the default value will be overridden on line 98.